src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -47,6 +47,13 @@
 // @todo #93:30min NHD calculation needs to take into account the method's
 //  visibility, which should be configurable and implemented after #101 is
 //  fixed.
+// @todo #68:30min SCOM has an impediment on issue #103: cannot currently
+//  be tested in MetricsTest when the resulting value is "NaN". Affected
+//  tests are: NoMethods, OneVoidMethodWithoutParams, WithoutAttributes,
+//  OneMethodCreatesLambda.
+// @todo #68:30min SCOM has an impediment on issue #114: the skeleton
```

Comment:
@llorllale Pls reprhase this puzzle; what exactly is the "impediment"? I think you want to say that a testcase for SCOM on class Bar cannot be added because of issue no. 114.